.NET/Microsoft.Recognizers.Text.DateTime/Italian/Parsers/ItalianDateParserConfiguration.cs

```diff
@@ -168,9 +169,8 @@ public int GetSwiftMonthOrYear(string text)
 
         public bool IsCardinalLast(string text)
         {
-            var trimmedText = text.Trim().ToLowerInvariant();
-            return trimmedText.Equals("dernière") || trimmedText.Equals("dernières") ||
-                    trimmedText.Equals("derniere") || trimmedText.Equals("dernieres");
+            var trimmedText = text.Trim();
+            return trimmedText.Equals("last");
```

Comment:
Shouldn't this be modified according to Italian? Is there a specific test case for this? I'm definitely not familiar with Italian, just a kind reminder.

Reply:
updated.